health-services/project-factory/src/server/utils/campaignMappingUtils.ts

```
@@ -509,7 +509,7 @@ async function processCampaignMapping(messageObject: any) {
             var completedResources: any = []
             var resources = [];
             for (const resourceDetailId of resourceDetailsIds) {
-                var retry = 75;
+                var retry: any = config?.retryUntilResourceCreationComplete;
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'let' instead.


(lint/style/noVar)